### ISSUE TITLE
B 1S T-Test: Test value isn't always used

### ISSUE THREAD

**Issue by jonathon-love** [jasp-stats/jasp-desktop#512]
hey @raviselker 

i think this [commit](https://github.com/jasp-stats/jasp-desktop/commit/b6ac0d861f2d451b4f8c3a1777d6692f3dd2dd3c) is incomplete. not all calls to ttestBF() have the test value specified (i'm guessing). see below (the wide prior, and the ultra-wide prior seem to be still tested against zero):

![screen shot 2015-03-17 at 10 00 41 pm](https://cloud.githubusercontent.com/assets/3240247/6697489/1ed36aa4-ccf1-11e4-855a-5e484cd2782c.png)

**Comment by jonathon-love** [comment]
this is a bit weird too. the column has a mean close to zero, but a one-tailed test asking if the mean is greater than -5 provides a BF10 of 0

what if i hand this one over to @quentingronau, as he's all over this stuff

![screen shot 2015-03-17 at 10 12 21 pm](https://cloud.githubusercontent.com/assets/3240247/6697717/b7934e0c-ccf2-11e4-90f0-d850707036c3.png)

**Comment by jonathon-love** [comment]
hey @quentingronau,

ravi wants me to give this to you.

**Comment by quentingronau** [comment]
Hi,

Yes, I will take a look. I think different things are going on here. A lot of Bayes factors are computed within the plotting functions, e.g., all the sequential BFs, the BFs under different prior widths, etc. The BFs that are displayed at the top of the robustness plot that you showed are all computed in the plotting function except the BF that corresponds to the user prior width. Also the posterior plot of the effect size needs to be adjusted when a different test value is specified. So, to sum this up, I need to change all the plotting functions, then the results should be correct for two-sided tests. Furthermore, we changed the way we compute one-sided BFs (we don't use ttestBF directly, but we use posterior samples because of issues with large BFs). At the moment it is not possble to specify a different test value in this function. I tried to adjust the code to make this possible and have sent it to Richard so that he can take a look whether it is correct, because I am a little bit afraid of making mistakes that lead to wrong output.

Quentin
